README.md

```diff
@@ -1,42 +1,103 @@
-# How does it look like?
+<h1 align="center">
+  <img src="./docs/assets/logo.png" alt="livechat design-system logo" /><br />
+  LiveChat Design System
+</h1>
 
-[See the Storybook](https://v1--613a8e945a5665003a05113b.chromatic.com/)
+<p align="center">
+  <a href="https://www.npmjs.com/@livechat/design-system-react-components">
+    <img alt="npm version react-components" src="https://img.shields.io/npm/v/@livechat/design-system-react-components.svg?label=react-components">
+  </a>
+  <a href="https://www.npmjs.com/@livechat/design-system-icons">
+    <img alt="npm version icons" src="https://img.shields.io/npm/v/@livechat/design-system-icons.svg?label=icons">
+  </a>
+  <a href="https://v1--613a8e945a5665003a05113b.chromatic.com/">
+    <img src="https://img.shields.io/static/v1?label=examples&message=storybook&color=ff4685">
+  </a>
+  <a href="https://github.com/livechat/design-system/actions/workflows/tests.yml">
+    <img src="https://img.shields.io/github/workflow/status/livechat/design-system/Tests/v1?label=tests">
+  </a>
+</p>
 
-# How to install?
+## Installation
 
-To run the project, first You need to use a the tool for managing the node version, which is convenient for you.
-We suggest using [Volta](https://volta.sh/).
+Run the following command using [npm](https://www.npmjs.com/) (or with you other favorite package manager, eg. [yarn](https://yarnpkg.com/)):
 
-Required version of node is `16.13.2` and npm `8.1.0`.
+```
+npm install @livechat/design-system-react-components @livechat/design-system-icons --save
+```
 
-Then, just run `npm install`.
+## Basic usage
 
-# How to use?
+Import the `CSS` directly into your project:
```

Comment:
As far as I know, we do not append CSS with components - I would stress here that this is a necessary step to work with components.